composer.json

```diff
@@ -24,11 +24,18 @@
 			"typo3/cms-composer-installers": true
 		}
 	},
-    "require": {
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "https://github.com/toujou/BunnyNet-PHP.git"
+		}
+	],
+	"require": {
 		"php": "^7.2 || ^8.0",
 		"guzzlehttp/guzzle": "^6.0 || ^7.0",
 		"typo3/cms-core": "^10.0 || ^11.0",
-		"doctrine/dbal": "~2.13.1"
+		"doctrine/dbal": "~2.13.1",
+		"toshy/bunnynet-php": "dev-master"
```

Comment:
Why not a regular version, like 2.3? Otherwise everyone using the proxycachemanager needs to add bunnynet-php as @dev dependency to their root composer.json.

Reply:
Further, we shouldn't use the API client for that one single guzzle call.